tasks.py

```diff
@@ -556,28 +558,92 @@ def test_api(c, product, backend=''):
     gmysql   = ['gmysql',     'gmysql-nodnssec-both',   'gmysql-nsec3-both',   'gmysql-nsec3-optout-both',   'gmysql-nsec3-narrow',   'gmysql_sp-both'],
     gpgsql   = ['gpgsql',     'gpgsql-nodnssec-both',   'gpgsql-nsec3-both',   'gpgsql-nsec3-optout-both',   'gpgsql-nsec3-narrow',   'gpgsql_sp-both'],
     gsqlite3 = ['gsqlite3', 'gsqlite3-nodnssec-both', 'gsqlite3-nsec3-both', 'gsqlite3-nsec3-optout-both', 'gsqlite3-nsec3-narrow'],
+    godbc_sqlite3 = [
+        'godbc_sqlite3-nodnssec'
+    ],
+    godbc_mssql = [
+        'godbc_mssql',
+        'godbc_mssql-nodnssec',
+        'godbc_mssql-nsec3',
+        'godbc_mssql-nsec3-optout',
+        'godbc_mssql-nsec3-narrow'
+    ],
 )
 
+godbc_mssql_credentials = {
+    'username' : 'sa',
+    'password' : 'SAsa12%%'
+}
+
+def setup_godbc_mssql(c):
+    c.run('''cat >> ~/.odbc.ini <<-__EOF__
+[pdns-mssql-docker]
+Driver=FreeTDS
+Trace=No
+Server=127.0.0.1
+Port=1433
+Database=pdns
+TDS_Version=7.1
+[pdns-mssql-docker-nodb]
+Driver=FreeTDS
+Trace=No
+Server=127.0.0.1
+Port=1433
+TDS_Version=7.1
+__EOF__''')
```

Comment:
I would prefer to use python for something like this. As an example putting the file contents in a variable called `odbc_config` and doing something like the following:

```python
with open("~/.odbc.ini", "w") as f:
    print(odbc_config, file=f)
```

Reply:
Or of course using `f.write(...)`

Reply:
Done